Modules/Sources/NetworkingCore/Validators/PlaceholderDataValidator.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+/// Placeholder implementation for `ResponseDataValidator`.
+///
+public final class PlaceholderDataValidator: ResponseDataValidator {
+    public init() {}
+
+    public func validate(data: Data) throws {
+        // no-op
+    }
+}
```

Comment:
This looks like a new file, but existed already

https://github.com/woocommerce/woocommerce-ios/blob/ae05dab3f00e2be53a264c2cbbd641a0a5630f5e/Networking/Sources/Core/Validators/PlaceholderDataValidator.swift#L1-L9